doc/docker/base-dev.yml

```diff
@@ -27,6 +27,7 @@ services:
      - HTTPCACHE_VARNISH_INVALIDATE_TOKEN
      - FASTLY_SERVICE_ID
      - FASTLY_KEY
+     - XDEBUG_CONFIG
```

Comment:
How those variable works when there is no default in .env file?

Reply:
XDEBUG_CONFIG variable is empty in a container and Xdebug works with default settings from its ini config file